plugin/trino-queryeventlistener/src/main/java/io/trino/plugin/queryeventlistener/QueryEventListener.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.queryeventlistener;
+
+import com.google.common.base.Strings;
+import io.airlift.log.Logger;
+import io.trino.spi.eventlistener.EventListener;
+import io.trino.spi.eventlistener.QueryCompletedEvent;
+import io.trino.spi.eventlistener.QueryCreatedEvent;
+import io.trino.spi.eventlistener.SplitCompletedEvent;
+
+import java.util.Map;
+
+public class QueryEventListener
+        implements EventListener
+{
+    private static final Logger logger = Logger.get(QueryEventListener.class);
+
+    public QueryEventListener(Map<String, String> config)
+    {
+    }
+
+    @Override
+    public void queryCreated(QueryCreatedEvent queryCreatedEvent)
+    {
+        if (Strings.isNullOrEmpty(queryCreatedEvent.getContext().getUser())) {
+            logger.error("EventType:'QueryCreate', QueryId:'%s', CreateTime:'%s', User:'%s', Schema:'%s', Catalog:'%s', SQLQuery'%s'",
```

Comment:
We should log an error that notifies us that the username is missing. The actual Event information of interest should be logged at the INFO level regardless (i.e. it shouldn't be in the `else` block below)